### ISSUE TITLE
Breadcrumb - Current page does not receive focus

### ISSUE THREAD

**Issue by orion-cengage** [cengage/react-magma#1247]
CENG_3296

Docs: https://react-magma.cengage.com/version/4.3.0/api/breadcrumb/

Success Criteria: https://www.w3.org/WAI/WCAG21/Understanding/name-role-value

Within the breadcrumb component, the current page is rendered as a `span` element. This element does not receive focus by default so assistive technology users may not reach it or understand it if they do.

Rather than using a span for the current page, use a link element with `href=""` and `aria-current="page"`.

**Comment by orion-cengage** [comment]
I reviewed the examples in the docs site and this looks great.
